controlplane/kubeadm/internal/workload_cluster.go

```diff
@@ -174,6 +178,37 @@ func (w *Workload) UpdateKubeletConfigMap(ctx context.Context, version semver.Ve
 		return err
 	}
 
+	// In order to avoid using two cgroup managers on the same machine, cgroupfs and systemd cgroup, starting from
+	// 1.21 image builder is going to configure containerd for using systemd cgroup, and the Kubelet configuration should be changed accordingly.
+	if version.GE(semver.MustParse("1.21.0")) {
```

Comment:
Could we parse this only once in a global variable? As in `var minVerKubeletSystemdDriver = ...`